zammad/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: zammad
-version: 10.1.1
-appVersion: 6.1.0-24
+version: '11.0.0'
```

can we remove the quotes here?
```suggestion
version: 11.0.0
```

---

Yes, removed.